### ISSUE TITLE
Thinking on the R_version check

### ISSUE THREAD

**Issue by zhengle-advantaseeds** [ficonsulting/RInno#24]
Currently, the .iss script is checking for an exact match of the R_version specified. Would it possible to check the R_version for a set? For example, instead of specifying R_version = '3.4.1', we can specify R_version >= '3.0.0'.

**Comment by Dripdrop12** [comment]
Another great idea @zhengle-advantaseeds! I think we could easily support this. I've just got to brush up on string and numeric conversions in Delphi Pascal to make sure the registry query can support this. We could implement it in the "inst/installation/code.iss" file.

**Comment by zhengle-advantaseeds** [comment]
Great! Thanks for the very nice package!

**Comment by trybik** [comment]
I also vote for that.

**Comment by Dripdrop12** [comment]
@trybik and @zhengle-advantaseeds. Can you explain some of your thoughts around supporting this? See my response to #34. The challenge with registry queries, is we would have to build all possible registry entries and query them specifically. That build process in Delphi Pascal would be dictated by the string entered in `create_app`. It is a decent amount of work, so I just want to make sure we discuss it more before developing it.

**Comment by trybik** [comment]
Here I have no idea what would check for current OS's R version involve, implementation-wise. I will again refer to idea that's given in the [R DESCRIPTION file specification](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Package-Dependencies):

> The ‘Depends’ field can also specify a dependence on a certain version of R — e.g., if the package works only with R version 3.0.0 or later, include ‘R (>= 3.0.0)’ in the ‘Depends’ field.

I think this is quite important because ATM if I wrote an app with most current R version, which was "3.4.1" and the user did not update his R (more frequent case), or has a newer R ver. "3.4.3", then either his OS's R becomes the newer one and he is forced to update all packages (quite annoying), or he just ends up with additional older R installation (much less annoying).

**Comment by zhengle-advantaseeds** [comment]
I agreed with @trybik. However, I did look into the Pascal script in the Inno, and it returns strings. Like @Dripdrop12 said, it would be a good amount of workload to implement 'R(>=3.0.0)'. Instead, could we modified the config file? For example, 

Step 1: in the RInno, use R_version as the minimum required version: for example, 0.0.5
Step 2: do the string editing in R: to automatically generate all the lower version: 0.0.4, 0.0.3, 0.0.2, 0.0.1.
Step 3: in the Inno, implement a for loop to loop over all the lower version in the registry check.

I am more familiar with R, so I can help with any workloads related to the R.
